src/misc/ErrorBanner/ErrorBanner.js

```diff
@@ -64,4 +71,10 @@ ErrorBanner.defaultProps = {
       'your favorite text editor.',
     toggledButtonText: 'Copied',
   },
+  error: {
+    comment: '',
+    detail: '',
+    status: '',
+    title: '',
+  },
```

Comment:
I think we can remove this default props if we have a DEFAULT_ERROR variable :slightly_smiling_face:

Reply:
I didn't know we could destructure an `undefined` value, but it seems to work :smile: